backend/app/migrator/disqus.go

```diff
@@ -73,7 +73,7 @@ func (d *Disqus) Import(r io.Reader, siteID string) (size int, err error) {
 
 	log.Printf("[DEBUG] imported %d comments to site %s", passed, siteID)
 
-	if failed > 0 && passed == 0 {
+	if passed == 0 {
```

Comment:
any particular reason why 0 imported should be an error? In addition, this code was forgiving a little bit by design to report as failed, only if nothing were imported.

Reply:
It looks like `if failed > 0 && passed == 0` never be executed until `if failed > 0 return ...` exists upper. Maybe swap this conditions? I mean first put `if failed > 0 && passed == 0` and then put `if failed > 0 return ...`
So when `failed` exists it should return error but differentiate cases when nothing were converted and parts were converted.

My verstion was if nothing were converted it should return error regardless of `failed` exists or nor. Yeah, I agree it is incorrect